<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewBugItem</class>
 <widget class="QDialog" name="CreateNewBugItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>220</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="ButtonBack">
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="ButtonNext">
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="ButtonCancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QStackedWidget" name="stackedCreateNewBug">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageDescribe">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>描述</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textDescribe"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageRemarks">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>备注</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textRemarks"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>优先级</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="radiohight">
             <property name="text">
              <string>高</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioMiddle">
             <property name="text">
              <string>中</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioLow">
             <property name="text">
              <string>低</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateNewBugItem</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
